### PR TITLE
Separate out the -ansi comment fixes.

### PR DESCRIPTION
**Pull request by DennisHeimbigner**
re: pull request https://github.com/Unidata/netcdf-c/pull/1242

This pr should be applied before pr https://github.com/Unidata/netcdf-c/pull/1242.
It fixes only the -ansi '//' comment problems. There may be some
slight conflicts with that other pr when it is applied, since in some
cases I converted #if 0...#endif to /* ... */

### CODE REVIEW

**Comment by t-b** [comment]
Looks good. I found a couple of more via the following git commit. Of course a view false positives are included.

```
$git grep -n '[^:/"]//' '*.c' '*.h'
examples/C/parallel_vara.c:34: *    // global attributes:
hdf4_test/tst_interops3.c:39:         //ERR;
include/XGetopt.h:40:#endif //XGETOPT_H
include/nc_tests.h:43:#define BAD_NAME "dd//d/  "
libdap2/obsolete/cdf4.c:511:    if(rankednode->nctype == NC_Dimension) continue; //ignore
libdap4/d4file.c:220:    fprintf(stderr,"\n/////////////\n");
libdap4/d4file.c:226:    fprintf(stderr,"\n/////////////\n");
libdap4/d4meta.c:1012:      //size = NCD4_computeTypeSize(builder,ftype);
libdap4/d4meta.c:1018:          //size = NCD4_computeTypeSize(builder,ftype);
libdap4/d4meta.c:1021:          //size = NCD4_computeTypeSize(builder,ftype);
libdap4/d4meta.c:1027:      //size = NCD4_computeTypeSize(builder,truetype);
libdap4/d4meta.c:1031:      //size = NCD4_computeTypeSize(builder,ftype);
libdispatch/json.c:34:/*//////////////////////////////////////////////////*/
libsrc/XGetopt.c:109://                       //
libsrc/XGetopt.c:110://                       // set some flag here
libsrc/XGetopt.c:111://                       //
libsrc/XGetopt.c:116://                       //
libsrc/XGetopt.c:117://                       // set some other flag here
libsrc/XGetopt.c:118://                       //
libsrc/XGetopt.c:123://                       //
libsrc/XGetopt.c:124://                       // do something with value here
libsrc/XGetopt.c:125://                       //
libsrc/XGetopt.c:139://           //
libsrc/XGetopt.c:140://           // check for non-option args here
libsrc/XGetopt.c:141://           //
libsrc/XGetopt.c:147:TCHAR      *optarg;                // global argument pointer
libsrc/XGetopt.c:148:int                optind = 0;     // global argv index
libsrc/XGetopt.c:184:           next++;         // skip past -
nc_test/nc_test.c:284:  /* NC_TEST(nc_enddef); *//* redundant */
nc_test/tst_inmemory.c:81:      fun = UNLIMITED ; // (2 currently)
nc_test/tst_inq_type.c:181:#endif // USE_HDF5
nc_test4/tst_ar4.c:27:  time = UNLIMITED ; // (1560 currently)
nc_test4/tst_ar4_3d.c:36:       time = UNLIMITED ; // (1560 currently)
nc_test4/tst_ar4_4d.c:37:       time = UNLIMITED ; // (1200 currently)
nc_test4/tst_xplatform.c:75:       }; // cmp_t
nc_test4/tst_xplatform.c:138:          }; // c
nc_test4/tst_xplatform.c:141:          }; // d
nc_test4/tst_xplatform.c:143:        // global attributes:
ncdap_test/test_cvt.c:129:    /* location of our target url: use file// to avoid remote
ncdump/XGetopt.c:109://                       //
ncdump/XGetopt.c:110://                       // set some flag here
ncdump/XGetopt.c:111://                       //
ncdump/XGetopt.c:116://                       //
ncdump/XGetopt.c:117://                       // set some other flag here
ncdump/XGetopt.c:118://                       //
ncdump/XGetopt.c:123://                       //
ncdump/XGetopt.c:124://                       // do something with value here
ncdump/XGetopt.c:125://                       //
ncdump/XGetopt.c:139://           //
ncdump/XGetopt.c:140://           // check for non-option args here
ncdump/XGetopt.c:141://           //
ncdump/XGetopt.c:147:TCHAR      *optarg;                // global argument pointer
ncdump/XGetopt.c:148:int                optind = 0;     // global argv index
ncdump/XGetopt.c:184:           next++;         // skip past -
ncdump/chunkspec.c:96:    /* Special rule: // is treated as equivalent to / */
ncdump/ncdump.c:1423:       printf("}; // %s\n", type_name);
ncdump/ncdump.c:1425:       printf("}; // ");
ncdump/ncdump.c:1651:         printf ("UNLIMITED ; // (%lu currently)\n",
ncdump/ncdump.c:1658:         printf ("UNLIMITED ; // (%u currently)\n",
ncdump/ncdump.c:1677:     printf ("UNLIMITED ; // (%u currently)\n",
ncdump/ncdump.c:1907:/*             printf ("} // group %s\n", group_name); */
ncdump/ncdump.c:1908:     printf ("} // group ");
ncdump/ncdump.h:24:    bool_t brief_data_cmnts; /* if true, put // comments in data section
ncdump/ncdump.h:29:    bool_t full_data_cmnts;  /* if true, put // comments in data section
ncdump/nctime0.c:248: *    time:actual_range = 51133., 76670. ; // "1940-01-01", "2009-12-01"
ncdump/tst_nans.c:50:    float fnan = NC_FNAN;/*(NC_INFINITE-NC_INFINITE);//0.f/0.f; */
ncdump/tst_nans.c:51:    double dnan = NC_DNAN;/*(NC_INFINITE-NC_INFINITE);//0.0/0.0; */
ncdump/tst_nans.c:52:    float fpinf = NC_FPINF;/*NC_INFINITE;//1.0f/0.0f; */
ncdump/tst_nans.c:54:    double dpinf = NC_DPINF;/*NC_INFINITE;//1.0/0.0; */
ncdump/vardata.c:300:/*     printf("  // %s(", vp->name); */
ncdump/vardata.c:301:    printf("  // ");
ncgen/XGetopt.c:109://                       //
ncgen/XGetopt.c:110://                       // set some flag here
ncgen/XGetopt.c:111://                       //
ncgen/XGetopt.c:116://                       //
ncgen/XGetopt.c:117://                       // set some other flag here
ncgen/XGetopt.c:118://                       //
ncgen/XGetopt.c:123://                       //
ncgen/XGetopt.c:124://                       // do something with value here
ncgen/XGetopt.c:125://                       //
ncgen/XGetopt.c:139://           //
ncgen/XGetopt.c:140://           // check for non-option args here
ncgen/XGetopt.c:141://           //
ncgen/XGetopt.c:147:TCHAR       *optarg;                // global argument pointer
ncgen/XGetopt.c:148:int         optind = 0;     // global argv index
ncgen/XGetopt.c:184:            next++;         // skip past -
ncgen/genf77.c:803:        /* Generate the data // statement */
ncgen3/XGetopt.c:109://                       //
ncgen3/XGetopt.c:110://                       // set some flag here
ncgen3/XGetopt.c:111://                       //
ncgen3/XGetopt.c:116://                       //
ncgen3/XGetopt.c:117://                       // set some other flag here
ncgen3/XGetopt.c:118://                       //
ncgen3/XGetopt.c:123://                       //
ncgen3/XGetopt.c:124://                       // do something with value here
ncgen3/XGetopt.c:125://                       //
ncgen3/XGetopt.c:139://           //
ncgen3/XGetopt.c:140://           // check for non-option args here
ncgen3/XGetopt.c:141://           //
ncgen3/XGetopt.c:147:TCHAR      *optarg;                // global argument pointer
ncgen3/XGetopt.c:148:int                optind = 0;     // global argv index
ncgen3/XGetopt.c:184:           next++;         // skip past -
ncgen3/genlib.c:1158: * "ab\ncd" would yield "'ab'//char(10)//'cd'".  The common
```

File: hdf4_test/tst_interops3.c [hunk 1]
```diff
@@ -36,7 +36,7 @@ check_inq_format(int ncid, int expected_format, int expected_extended_format, in
       if (nc_inq_format_extended(ncid, NULL, &mode)) ERR;
       if (mode != expected_mode) {
          printf("expected_mode %x mode %x\n", expected_mode, mode);
-         //ERR;
+         /*ERR;*/
```
Comment (t-b) [inline hdf4_test/tst_interops3.c]: Looks good!

**Comment by DennisHeimbigner** [comment]
its false positive; I forgot to kill the obsolete directory.

**Comment by WardF** [comment]
Looks good and fixed the OSX issue, I’ll merge this as soon as the tests complete.

**Comment by WardF** [comment]
I will take a look at those next, thanks!

**Pull request merged**